observability/logging/logging-6.1/log6x-about-6.1.adoc

```diff
@@ -0,0 +1,65 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="log6x-about-6.1"]
+= Logging 6.1
+context: logging-6x-6.1
+
+toc::[]
+
+The `ClusterLogForwarder` custom resource (CR) is the central configuration point for log collection and forwarding.
+
+== Inputs and Outputs
+
+Inputs specify the sources of logs to be forwarded. Logging provides built-in input types: `application`, `receiver`, `infrastructure`, and `audit`, which select logs from different parts of your cluster. You can also define custom inputs based on namespaces or pod labels to fine-tune log selection.
+
+Outputs define the destinations where logs are sent. Each output type has its own set of configuration options, allowing you to customize the behavior and authentication settings.
+
+
+== Receiver Input Type
+The receiver input type enables the Logging system to accept logs from external sources. It supports two formats for receiving logs: `http` and `syslog`.
+
+The `ReceiverSpec` defines the configuration for a receiver input.
+
+== Pipelines and Filters
+
+Pipelines determine the flow of logs from inputs to outputs. A pipeline consists of one or more input refs, output refs, and optional filter refs. Filters can be used to transform or drop log messages within a pipeline. The order of filters matters, as they are applied sequentially, and earlier filters can prevent log messages from reaching later stages.
+
+== Operator Behavior
+
+The Cluster Logging Operator manages the deployment and configuration of the collector based on the `managementState` field of the `ClusterLogForwarder` resource:
+
+- When set to `Managed` (default), the operator actively manages the logging resources to match the configuration defined in the spec.
+- When set to `Unmanaged`, the operator does not take any action, allowing you to manually manage the logging components.
+
+== Validation
+Logging includes extensive validation rules and default values to ensure a smooth and error-free configuration experience. The `ClusterLogForwarder` resource enforces validation checks on required fields, dependencies between fields, and the format of input values. Default values are provided for certain fields, reducing the need for explicit configuration in common scenarios.
+
+[id="quick-start_{context}"]
+== Quick start
+
+OpenShift Logging supports two data models:
+
+* ViaQ (General Availability)
+* OpenTelemetry (Technology Preview)
+
+You can select either of these data models based on your requirement by configuring the `lokiStack.dataModel` field in the `ClusterLogForwarder`. ViaQ is the default data model when forwarding logs to LokiStack. For more information on OpenTelemetry, see xref:../../../observability/logging/logging-6.1/log6x-opentelemetry-data-model-6.1.adoc#log6x-opentelemetry-data-model-6.1[OpenTelemetry data model].
+
+[NOTE]
+====
+In future releases of OpenShift Logging, the default data model will change from ViaQ to OpenTelemetry.
```

Comment:
> In future releases of OpenShift Logging, the default data model will change from ViaQ to OpenTelemetry.

This is:
- not an approved short name for logging 
- [explicitly forbidden by the supplementary style guide. ](https://redhat-documentation.github.io/supplementary-style-guide/#statements-about-the-future)